src/Rules.Framework/ConditionNodes/IValueConditionNode.cs

```diff
@@ -0,0 +1,30 @@
+namespace Rules.Framework.ConditionNodes
+{
+    using Rules.Framework;
+
+    /// <summary>
+    /// Defines the interface contract for a condition node based on a value comparison.
+    /// </summary>
+    public interface IValueConditionNode : IConditionNode
+    {
+        /// <summary>
+        /// Gets the condition node type.
+        /// </summary>
+        string ConditionType { get; }
+
+        /// <summary>
+        /// Gets the condition node data type.
+        /// </summary>
+        DataTypes DataType { get; }
+
+        /// <summary>
+        /// Gets the condition's operand.
+        /// </summary>
+        public object Operand { get; }
```

Comment:
This public can be removed since the class is already public

Reply:
Will be resolved on #171.